README.md

```diff
@@ -49,375 +45,258 @@ NV-Ingest supports the following file types:
+```
 
-To interact from the host, you'll need a Python environment and install the client dependencies:
+You can see the extracted text that represents the content of the ingested test document.
+```shell
+Testing Document
 
-```bash
-# conda not required, but makes it easy to create a fresh python environment
-conda env create --name nv-ingest-dev --file ./conda/environments/nv_ingest_environment.yml
-conda activate nv-ingest-dev
+A sample document with headings and placeholder text
 
-cd client
-pip install .
+Introduction
 
-# When not using Conda, pip dependencies for the client can be installed directly via pip. Pip based installation of
-# the ingest service is not supported.
-cd client
-pip install .
-```
+This is a placeholder document that can be used for any purpose. It contains some headings and some placeholder text to fill the space. The text is not important and contains no real value, but it is useful for testing. Below, we will have some simple tables and charts that we can use to confirm Ingest is working as expected.
 
-> [!NOTE]
-> Interacting from the host depends on the appropriate port being exposed from the nv-ingest container to the host as defined in [docker-compose.yaml](docker-compose.yaml).
->
-> If you prefer, you can disable exposing that port, and interact with the nv-ingest service directly from within its container.
->
-> To interact within the container:
-> ```
-> docker exec -it nv-ingest-nv-ingest-ms-runtime-1 bash
-> ```
-> You'll be in the `/workspace` directory, which has `DATASET_ROOT` from the .env file mounted at `./data`. The pre-activated `morpheus` conda environment has all the python client libraries pre-installed:
-> ```
-> (morpheus) root@aba77e2a4bde:/workspace#
-> ```
->
-> From the bash prompt above, you can run nv-ingest-cli and Python examples described below.
+Chart 1
 
-### Step 3: Ingesting Documents
+This chart shows some gadgets, and some very fictitious costs.
 
-You can submit jobs programmatically in Python or via the [NV-Ingest CLI](nv-ingest_cli.md).
 
-In the below examples, we are doing text, chart, table, and image extraction:
 
-- **extract_text** — Uses [PDFium](https://github.com/pypdfium2-team/pypdfium2/) to find and extract text from pages.
-- **extract_images** — Uses [PDFium](https://github.com/pypdfium2-team/pypdfium2/) to extract images.
-- **extract_tables** — Uses [YOLOX](https://github.com/Megvii-BaseDetection/YOLOX) to find tables and charts. Uses [PaddleOCR](https://github.com/PaddlePaddle/PaddleOCR) for table extraction, and [Deplot](https://huggingface.co/google/deplot) and CACHED for chart extraction.
-- **extract_charts** — (Optional) Enables or disables Deplot and CACHED for chart extraction.
+\begin{tabular}{ccc}
+**Animal** & **Activity** & **Place**\\
+Giraffe & Driving a car & At the beach\\
+Lion & Putting on sunscreen & At the park\\
+Cat & Jumping onto a laptop & In a home office\\
+Dog & Chasing a squirrel & In the front yard\\
+\end{tabular}
```

Comment:
Nit, this looks like Eclair's output (LaTex format). Was it intentional, or do we actually want Paddle's markdown format? It looks like the quickstart doc also has a couple in LaTeX format, so I was curious.

Reply:
good catch. I had an eclair run in my milvus.db file from a prior test run

Reply:
fixed